src/Logging.AzureAppServices/test/LoggerBuilderExtensionsTests.cs

```diff
@@ -24,53 +24,106 @@ public LoggerBuilderExtensionsTests()
         }
 
         [Fact]
-        public void BuilderExtensionAddsSingleSetOfServicesWhenCalledTwice()
+        public void BuilderExtensionAddsSingleSetOfServicesWhenCalledTwiceOrig()
         {
             var serviceCollection = new ServiceCollection();
-            serviceCollection.AddLogging(builder => builder.AddAzureWebAppDiagnostics(_appContext));
+            serviceCollection.AddLogging(builder => builder.AddAzureWebAppDiagnostics(_appContext, null));
```

Comment:
This shouldn't be needed, the overload should set the prefix correctly. If this doesn't work, there is a problem with back compatibility.
```suggestion
            serviceCollection.AddLogging(builder => builder.AddAzureWebAppDiagnostics(_appContext));
```